### DISCUSSION TITLE
How to override button in odoo 14?

### DISCUSSION THREAD

**Discussion by it12uw** [OCA/odoo-community.org#134] [Q&A]
i want to custom the Save & New button in odoo 14, i want to custome this button because i want everytime i click the button the form and the view for layout (radio button) is not refresh .

![Screenshot 2024-01-03 092336](https://github.com/OCA/odoo-community.org/assets/135787412/96f93aaf-941f-4ef0-81e6-5a045750f590)

thankyou

**Comment by DealTechSrl** [comment]
Seems like a transient model, so when you click save & close/new, it will run the ORM methods like create or write.
